### ISSUE TITLE
System refuses to boot while using multipath for the rootfs since 759d9f3

### ISSUE THREAD

**Issue by mrc0mmand** [systemd/systemd#10882]
**systemd version the issue has been seen with**
> v239 (latest master)

**Used distribution**
 > CentOS 7.5

This is a follow-up issue for discussion in https://github.com/systemd/systemd-centos-ci/issues/18#issuecomment-440982075

For some reason the system refuses to boot since 759d9f3 (according to bisect). The target machine uses multipath for the root filesystem.

Logs (will try to provide debug logs as well):
[dmesg](http://sprunge.us/5mA5w9)
[journal](http://sprunge.us/RMJf7k)

**Comment by yuwata** [comment]
Ah, I notice one bug. I will submit a fix.

**Comment by evverx** [comment]
Oh, come on :-) Let's revert something :-)

**Comment by yuwata** [comment]
@mrc0mmand Could you try #10886?

**Comment by mrc0mmand** [comment]
Will do!

**Comment by yuwata** [comment]
Hmm, the PR fixes a bug in the commit, but it seems that your system also affected by #10754. Please also try 'udev.children_max=1' or relevant options if the system still does not boot.

**Comment by mrc0mmand** [comment]
The system boots even without the `udev.children_max=1` option, thank you! I see some udev-related errors in the journal - are those from the mentioned PR or it's a completely different issue?

```
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2937]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2848]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2846]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2774]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2702]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2950]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2822]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2870]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2773]: Failed to wait spawned command '/sbin/multipath -T 1542930287: -c /dev/dm-1': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2937]: Failed to wait spawned command '/bin/systemd-detect-virt': Input/output error
Nov 22 23:44:51 n15.pufty.ci.centos.org systemd-udevd[2873]: Failed to wait spawned command '/sbin/multipath -T 1542930287: -c /dev/dm-2': Input/output error

```

**Comment by yuwata** [comment]
Thank you for testing!

> I see some udev-related errors

I've never seen such logs. Maybe a new bug...

**Comment by evverx** [comment]
In the meantime, I've just turned the CentoCI webhook on and force-pushed https://github.com/systemd/systemd/pull/10885 to trigger it to see if it works. Apparently, it doesn't because https://ci.centos.org/ghprbhook/ responded with 500. I guess a new job should be created as described in https://wiki.centos.org/QaWiki/CI/GithubIntegration to make it work.

**Comment by mrc0mmand** [comment]
The agent script needs some little tweaking as well. I'd play with the webhooks as well, but I can't. Will try to shape the CentOS CI into a working state tomorrow though.

**Comment by evverx** [comment]
It turns out that @centos-ci was blocked. I unblocked it and invited it as a collaborator when I noticed that we have @systemd-centos-ci, which most likely has something to do with CentOS CI. I'm afraid I have no idea what's up with all this so maybe someone else could help. I canceled the invitation and turned the hook off again.
